central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl.go

```diff
@@ -640,7 +651,14 @@ func (m *managerImpl) getFixableCVEDeferrals() ([]*storage.VulnerabilityRequest,
 				fixableReqs = append(fixableReqs, request)
 			}
 		} else if expiryType == storage.RequestExpiry_ALL_CVE_FIXABLE {
-			results, err := m.imageCVEs.Search(ctx, fixableQuery)
+
+			var results []search.Result
+			var err error
+			if features.FlattenCVEData.Enabled() {
+				results, err = m.imageCVE2.Search(ctx, fixableQuery)
```

Comment:
Similar to above.  Does this query need to return a grouping or just all the instances of a CVE?  Again think about the case where an image has multiple occurrences of the same CVE.

Reply:
CVEs should be deduped here since resulrs are iterated over to get CVE ids using search.ResultsToIDs and Ids are stored in a set to count total number of CVEs